docs/guides/online-web-audio.md

```diff
@@ -0,0 +1,221 @@
+---
+layout: default
+---
+
+This tutorial will guide you through the steps of using Meyda to analyze audio
+in the context of the [Web Audio API][webaudioapi]. The Web Audio API is a set
+of tools available in most major browsers that let you build immersive audio
+experiences in the web.
+
+If you run into any difficulties, please help us out by [reporting an issue]
+[new-issue] so that we can clarify this document for everyone.
+
+Today we'll build a website that plays an audio file, and shows the user the
+volume of the audio as it plays. Lets get started!
```

Comment:
The feature I'm using is `rms`, and I said "volume" because I didn't want to confuse the reader by making them read and understand what rms is. I think "energy" would confuse the reader with [our energy feature](https://github.com/meyda/meyda/wiki/Audio-Features#energy) - how about "level"?

Reply:
Ah yes, I see where you're coming from. I just got confused because there was a volume slider that was changing but without any change in volume. Level could work, also strength maybe?

Reply:
I just pushed a copy that says "level", and I think the average developer would be a little more familiar with calling it level rather than strength - searching for "audio level" gives me a lot more relevant results than searching for "audio strength".